src/os/ObjectStore.h

```diff
@@ -1738,6 +1746,7 @@ class ObjectStore {
         _op->oid = _get_object_id(oid);
         _op->expected_object_size = expected_object_size;
         _op->expected_write_size = expected_write_size;
+	_op->alloc_hint_flags = flags;
```

Comment:
Don't know what's the purpose for use_tbl option but shouldn't we encode flags for that flow too?